eZ/Publish/Core/Repository/RoleService.php

```diff
@@ -509,38 +542,19 @@ public function publishRoleDraft(APIRoleDraft $roleDraft)
         }
     }
 
-    /**
-     * Deletes a policy.
-     *
-     * Used by {@link removePolicy()} and {@link deletePolicy()}
-     *
-     * @param APIPolicy $policy
-     *
-     * @throws \Exception
-     */
-    protected function internalDeletePolicy(APIPolicy $policy)
```

Comment:
Please avoid moving methods if the diff is big, it's difficult to see what has changed.